modules/dns_common/dependency_injection/dependencies.rb

```diff
@@ -3,9 +3,8 @@ module DependencyInjection
     class Dependencies
       extend Proxy::Log
       extend Proxy::DependencyInjection::Wiring
+
       def self.container_instance
-        logger.warn('Proxy::Dns::DependencyInjection::Dependencies class has been deprecated and will be removed in future versions. '\
-          'Please use ::Proxy::Dns::YourPlugin#load_dependency_injection_wirings instead.')
```

Comment:
Are you sure it's in heavy use? Can you point me to some use of it?

Reply:
Well not heavy use, that was a bad grep. Smart Proxy IPAM plugin does appear to use it somehow, but I am not sure the DI stuff is unnecessary complex. I have no intentions to be fixing code that makes use of it, so I am removing this from my patchset.

Reply:
No, that's just regular dependency injection. IPAM doesn't use this class and you can safely drop it.